platform/ios/PolyPodApp/Extensions/UserDefaults+Utils.swift

```diff
@@ -1,26 +1,19 @@
 import Foundation
+import PolyPodCoreSwift
 
 extension UserDefaults {
-    enum Keys: String, CaseIterable {
-        case resetUserDefaults
-        case firstRun
-        case skipSecurity
+    private enum MigratedKeys: String, CaseIterable {
         case lastUpdateNotificationId
         case lastUpdateNotificationState
-        case updateNotificationMockId
-        case disableAuthCheck
-        case authSetUp
-        case didConfigureAuth
-        case showDeveloperFeaturesId
     }
     
     func reset() {
         Keys.allCases.forEach { removeObject(forKey: $0.rawValue) }
     }
     
-    // Currently we have project wide file protection enabled. 
+    // Currently we have project wide file protection enabled.
     // So, data protection is disabled for particular components.
-    // Removing project wide data protection, and then enable when needed 
+    // Removing project wide data protection, and then enable when needed
```

Comment:
Think this was automatic - guess I'll leave it in if nobody minds, otherwise the next change is gonna trigger this I suppose.